.github/workflows/trigger-anaconda.yml

```diff
@@ -19,16 +19,18 @@ jobs:
     permissions:
       contents: read
       statuses: write
-    container: registry.fedoraproject.org/fedora:rawhide
+    container:
+      image: ghcr.io/cockpit-project/tasks
```

Comment:
Why not a more usual stable fedora image like 40?

Reply:
Good question! My impression was that we are moving towards running as much as possible inside our task container, so that we need to worry about as few containers as possible.  And this workflow calls `bots/tests-trigger`, and our task container is meant to execute bots code like this, no?

So, my impression is task-container == some stable thing plus dependencies required by our bots. Is that not so?

Reply:
If we want to run this in the tasks container, then I feel like we should make sure we already have the dependencies that this thing needs present in the tasks container.

So like:
 - use tasks container; or
 - standard image + `dnf install` some stuff

pick one.

Reply:
Alright: #20478